Week3/task05ObjectProperties.js

```diff
@@ -0,0 +1,14 @@
+let person = {
+    firstname:"John",
+    lastname:"Doe",
+    age:50,
+    eyecolor:"blue"
+};
+function showProVa() {
+    console.log('Parameters of an object is '+Object.getOwnPropertyNames(person));
+    console.log('The actual values of the parameters is:');
+    let x;
+    for (x in person) {
+    console.log(person[x]);
```

Comment:
Hmm.. I think a better way to do this would be like this:

```
function showPropValue(obj) {
  for (const prop in obj) {
    console.log(obj[prop]);
  }
}

showPropValue(person);
```

What do you think? Can you see some advantages of writing a function like this?

Reply:
Well, I think I wrote pretty the same function. The only difference is that my function shows not only values but names of properties also. As it was asked in a task.